schedule/qam/common/qam-dracut-basic@64bit.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: qam-dracut-basic
+description:
+vars:
+  COLLECT_COREDUMPS: '1'
+  EXTRATEST: prepare
```

Comment:
why EXTRATEST prepare ? existing one is darcut an with yaml is this variable not needed

Reply:
Removed